.github/workflows/ci.yaml

```diff
@@ -31,15 +31,11 @@ jobs:
         run: yarn test
       - name: Build
         run: yarn build
-  chromatic-deployment:
-    runs-on: ubuntu-latest
-    # Job steps
-    steps:
-      - uses: actions/checkout@v1
-      - name: Install dependencies
-        run: yarn
```

Comment:
there didn't really seem to be a need to separate this into two jobs and run yarn install again so I tucked it onto the previous job